feat(login-page): initial design guidance (#21)
vmware-clarity/clarity-guidance: 1017-login-page-design-guidance.md

---
category: component design guidance
state: draft
created: 2024-10-18
updated: 2024-10-21
---

# Login Page Component

> The [Login Page](https://clarity.design/documentation/login) is a predefined form for applications that require authentication.

Reviewer: ```suggestion
The [login page](https://clarity.design/documentation/login) is a predefined form for applications that require authentication.
```

> The [login page](https://clarity.design/documentation/login) is a predefined form for applications that require authentication.

Reviewer 2: ```suggestion
The [login page](https://clarity.design/documentation/login) is a predefined form designed for applications that require user authentication, providing a streamlined interface for entering credentials.
```

## Guidance

> - **Must** use Login Page component for authentication flows to maintain consistency across the applications.

Reviewer: ```suggestion
- **Must** use login page component for authentication flows to maintain consistency across the applications.
```

> - **Must** NOT use an asterick (`*`) in the input fields of a Login Page.

Reviewer: ```suggestion
- **Must not** use an asterisk (`*`) in the input fields of a login page.
```

> - **Must not** use an asterisk (`*`) in the input fields of a login page.

Reviewer: ```suggestion
- **Must** follow the [password guidance](/1019) for the password input.
```

Let's not repeat guidance when we can just link to it.

> - **Must** follow the [password design guidance](/1019) for the password input.

Reviewer 2: ```suggestion
- Design teams **must** follow the [password design guidance](https://github.com/1019) for the password input to ensure security and usability in the authentication process.
```

> - The username/password validation message **must not** call out whether the error occurred in the username or password field.

Reviewer 2: ```suggestion
- Design teams **must not** specify whether the error occurred in the username or password field in the validation message, as this can compromise security and lead to unauthorized access attempts.
```
